src/pysocha/build.py

```diff
@@ -164,6 +170,33 @@ def generate_blog_posts(config: dict):
                                                 'title': 'Posts for author: ' + author,
                                                 'posts': posts}))
 
+    sorted_posts.sort(key=lambda post: post['PostedDate'], reverse=True)
+    if blog_config['atomFeeds']:
+        atom_feed = ""
+        atom_feed += ATOM_HEADER
+        atom_elements = {
+            'siteTitle': config['siteTitle'],
+            'siteAddress': config['siteAddress'],
+            'author': first_author,
+            'updatedDate': sorted_posts[0]['PostedDate'].isoformat(),
```

Comment:
Will `author` get formatted to be an empty string if there are no blog posts to set `first_author`?
Similarly for `updatedDate`, `sorted_posts` isn't guaranteed to have elements, correct?

Reply:
`sorted_posts` may be an empty list, but we're only here if they asked for an atom feed, so this would error out LONG before this. The configuration not making sense is fine. I'd love to give more helpful error messages in case of failures when I get to what those will be though.